src/Exchange.jl

```diff
@@ -126,7 +148,7 @@ function update_max_stepsize!(mc_state::MCState, n_update, a, v, r; min_acc = 0.
     acc_rate = mc_state.count_atom[2] / (n_update * a)
     if acc_rate < min_acc
         mc_state.max_displ[1] *= 0.9
-    elseif acc_rate > max_acc
+    elseif acc_rate > max_acc && mc_state.max_displ[1]<=10.
```

Comment:
What is this extra condition for? What is the value 10.?

Reply:
I think I added it because max displacement gets really large in gas phase, messing up with the acceptance rate, perhaps it's not needed